src/Microsoft.Azure.WebJobs.Host/Common/CommonCollectorBinding.cs

```diff
@@ -6,7 +6,7 @@
 using Microsoft.Azure.WebJobs.Host.Bindings;
 using Microsoft.Azure.WebJobs.Host.Protocols;
 
-namespace Microsoft.Azure.WebJobs.ServiceBus
+namespace Microsoft.Azure.WebJobs.Host
```

Comment:
I've removed Common, and split out between the existing Bindings and Triggers folders. 

---
In reply to: [54930656](https://github.com/Azure/azure-webjobs-sdk/pull/643#discussion_r54930656) [](ancestors = 54930656)